docs/getting-started/mixed/index.md

```diff
@@ -0,0 +1,233 @@
+# Getting Started - Multiple Providers (Service Mesh Interface and NGiNX Ingress)
+
+!!! important
+    Available since v1.1
```

Comment:
can you please update this to 1.2

Reply:
@harikrongali rebased to master, re-tested, etc and looks good against master, u think we ain't in time for 1.1 ? if u are sure we can't make it for 1.1 I will change it for 1.2, please confirm.

Reply:
yes..1.1 release cut is already made last week and only bugs or regressions will go in..no features...next features will be part of 1.2

Reply:
done